petab/observables.py

```diff
@@ -55,7 +55,7 @@ def get_observable_df(
     except KeyError:
         raise KeyError(
             f"Observable table missing mandatory field {OBSERVABLE_ID}."
-        )
+        ) from None
```

Comment:
`as err`, `from err`?

Reply:
I'd prefer `from None`. We only want this simple message here without the pandas error on top.